docs/bake-reference.md

```diff
@@ -10,20 +10,126 @@ You can define your Bake file in the following file formats:
 - JSON
 - YAML (Compose file)
 
+### Lookup order
+
 By default, Bake uses the following lookup order to find the configuration file:
 
-1. `docker-bake.override.hcl`
-2. `docker-bake.hcl`
-3. `docker-bake.override.json`
-4. `docker-bake.json`
-5. `docker-compose.yaml`
-6. `docker-compose.yml`
+1. `compose.yaml`
+2. `compose.yml`
+3. `docker-compose.yml`
+4. `docker-compose.yaml`
+5. `docker-bake.json`
+6. `docker-bake.override.json`
+7. `docker-bake.hcl`
+8. `docker-bake.override.hcl`
 
-Bake searches for the file in the current working directory.
 You can specify the file location explicitly using the `--file` flag:
 
 ```console
-$ docker buildx bake --file=../docker/bake.hcl --print
+$ docker buildx bake --file ../docker/bake.hcl --print
+```
+
+If you don't specify a file explicitly, Bake searches for the file in the
+current working directory.
+
+### Conflicts
+
+If more than one Bake file is found, all files are merged into a single
+definition. Files are merged according to the [lookup order](#lookup-order).
```

Comment:
Ah I see you talk about it here but maybe we should also stipulate it in lookup order section?